frontend/components/Home.jsx

```diff
@@ -168,9 +174,7 @@ function Home__RenderFunc(props,ref) {
                                     "Home__text__k3LxN"
                                 )}
                             >
-                                {
-                                    "Canvas App Explorer is a collection of resources to assist the instructor in using the best tools available for you and your students. "
-                                }
+                                {"Find the Best Tools for Your Class and Students"}
```

Comment:
We want to eventually turn this into a slot, I think we have an issue for that though.